test-frame-metrics-collector/src/main/java/io/skodjob/testframe/MetricsCollector.java

```diff
@@ -205,7 +205,7 @@ protected void setExec(Exec exec) {
         this.exec = exec;
     }
 
-    private synchronized KubernetesClient getKubeClient() {
+    synchronized KubernetesClient getKubeClient() {
```

Comment:
maybe stupid question, but is the removal only due to test or?

Reply:
y, but I have exposed it only to package-protected ;). So there is no change to the public API. I don't want to use ugly reflection within the tests.